### DISCUSSION TITLE
Ldap

### DISCUSSION THREAD

**Discussion by gerichkoeke** [emrahcom/galaxy#2] [General]
good afternoon, is it possible to configure the galaxy for ldap authentication? Can I leave it to just 1 fixed server?

**Comment by emrahcom** [comment]
Hi @qhz-redes,

`Galaxy` uses [Ory Kratos](https://github.com/ory/kratos) as identity provider. During the setup, the installer installs `Ory Kratos` with a `Postgresql` backend into the server and there isn't a built-in support for `LDAP` authentication by default.

But [Ory Kratos](https://github.com/ory/kratos)  is a very flexible software and it is possible to integrate it with other identity providers. You may check its documents. I don't know if it is possible to integrate it with an `LDAP` backend and I never tried before...

**Comment by gerichkoeke** [comment]
Thanks for the feedback, I'll take a look at the Ory Kratos documentation,
if successful I'll come back with the configuration. Another doubt does the
galaxy have an administration panel? I haven't found a way to access the
dashboard. I'm waiting.

Em ter., 21 de fev. de 2023 às 04:30, emrah ***@***.***>
escreveu:

> Hi @qhz-redes <https://github.com/qhz-redes>,
>
> Galaxy uses Ory Kratos <https://github.com/ory/kratos> as identity
> provider. During the setup, the installer installs Ory Kratos with a
> Postgresql backend into the server and there isn't a built-in support for
> LDAP authentication by default.
>
> But Ory Kratos <https://github.com/ory/kratos> is a very flexible
> software and it is possible to integrate it with other identity providers.
> You may check its documents. I don't know if it is possible to integrate it
> with an LDAP backend and I never tried before...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emrahcom/galaxy/discussions/2#discussioncomment-5060865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A55SRY7YZJCOZDWYTU6XK2DWYRVLBANCNFSM6AAAAAAVCFOOOU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Gabriel Erich
Skype: gabriel.erich.koeke
Celular: (48) 9 9184 - 8242

**Reply by emrahcom** [reply]
Hi again,

If you ask for an admin panel to manage user accounts, no...
